democracylab/settings.py

```diff
@@ -402,4 +402,6 @@ def read_connection_config(config):
 
 BOARD_OF_DIRECTORS = os.environ.get('BOARD_OF_DIRECTORS', '')
 
-FAVICON_PATH = os.environ.get('FAVICON_PATH', 'https://d1agxr2dqkgkuy.cloudfront.net/img/favicon.png')
\ No newline at end of file
+FAVICON_PATH = os.environ.get('FAVICON_PATH', 'https://d1agxr2dqkgkuy.cloudfront.net/img/favicon.png')
+
+BLOG_URL = os.environ.get('BLOG_URL', 'https://blog.democracylab.org')
```

Comment:
Let me know if this is how we want it to fall back if no BLOG_URL is declared manually. My thinking was we could always retarget the blog.democracylab.org cname record to go wherever we want so it'd be our backup option and the blog link would always be visible and working. On the other hand if we want this to be a "show blog link _only if_ BLOG_URL is declared, otherwise hide it entirely" I'd have to rewrite this.

Reply:
Actually let's not do this default blog url, since there might be cases where we want the blog to be taken off entirely.